doc/man/man7/elektra-backends.7

```diff
@@ -1,14 +1,14 @@
 .\" generated with Ronn-NG/v0.10.1
 .\" http://github.com/apjanke/ronn-ng/tree/0.10.1.pre1
-.TH "ELEKTRA\-BACKENDS" "7" "February 2019" ""
+.TH "ELEKTRA\-BACKENDS" "7" "November 2021" ""
 .SH "NAME"
 \fBelektra\-backends\fR \- the backend concept
 .P
 Elektra has introduced \fBbackends\fR to support the storage of key databases in different formats\. Elektra abstracts configuration so that applications can receive and store settings without carrying information about how and where these are actually stored\. It is the purpose of the backends to implement these details\.
 .P
 Since Elektra 0\.8 a backend is composed of many plugins\.
 .SH "MULTIPLE PLUGINS"
-It's clear that too many features in one backend is problematic\. It introduces unwanted external dependences and leads to less portable backends\. Many different aspects clutter the code making the backends unmaintainable\. Features of other backends cannot be taken with ease because they are interwoven with other code\.
+It's clear that too many features in one backend is problematic\. It introduces unwanted external dependencies and leads to less portable backends\. Many different aspects clutter the code making the backends unmaintainable\. Features of other backends cannot be taken with ease because they are interwoven with other code\.
```

Comment:
```suggestion
It's clear that too many features in one backend are problematic\. It introduces unwanted external dependencies and leads to less portable backends\. Many different aspects clutter the code making the backends unmaintainable\. Features of other backends cannot be taken with ease because they are interwoven with other code\.
```